SQL-assignment.sql

```diff
@@ -0,0 +1,70 @@
+use baseball;
+
+# create a table that has a column called DATEFIDD whcih calculate
+# the differecnes day between a specific date and local_date in game table
+drop table if exists batting_avg_wDATEDIFF ;
+CREATE table batting_avg_wDATEDIFF (
+batter int unsigned not null,
+hit int unsigned not null,
+atbat int unsigned not null,
+local_date datetime not null,
+diff_day int not null
+) engine=InnoDB default charset latin1;
```

Comment:
Definitely not necessary to change, but you could combine the CREATE and insert into one shorter statement by selecting directly into a new table.

Reply:
Very true. Since you're basing your SQL off of other tables and not populating it with RAW data I would say that would be preferred. But, it's not the end of the world. If you're just learning SQL this is a good way to get learn it.